app.py

```diff
@@ -0,0 +1,21 @@
+from apig_wsgi import make_lambda_handler
+from bottle import default_app, template, static_file, route
+
+app = default_app()
+
+lambda_handler = make_lambda_handler(app)
+
+
+@route('/static/<path:path>')
+def static(path):
+    return static_file(path, "./static")
+
+
+@route('/<path:re:.*>')
+def other_paths(path):
+    return template('index.html')
```

Comment:
The HTTP status should be 503 Service Unavailable. Should that be configured here, or does it happen in the ALB layer?

Reply:
It needs to be configured here. I've changed the response to return status code 503 for the get and post. The static endpoint which serves the css and the images still needs to return 200 so I've left that.